conda-recipe/meta.yaml

```diff
@@ -25,6 +25,7 @@ requirements:
     - numpy
     - scipy
     - pyepics
+    - p4p
```

Comment:
Dependencies of p4p seems reasonably light, should be OK to add as a req for linux, windows issues notwithstanding:
```
dependencies:
  - cython
  - epicscorelibs 7.0.4.99.*
  - libgcc-ng >=9.4.0
  - libstdcxx-ng >=9.4.0
  - nose >=1.1.2
  - numpy >=1.19.5,<2.0a0
  - ply
  - python >=3.9,<3.10.0a0
  - python_abi 3.9.* *_cp39
```